force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -322,6 +393,22 @@ public with sharing class FlowControlLoopsCollections {
      */
     public static Integer getSalary(String employeeName) {
         // Initialize a Map
+        Map<String, Integer> employeeSalaries = new Map<String, Integer>();
+        employeeSalaries.put('John Doe' , 50000);
+        employeeSalaries.put('Jane Smith', 60000);
+        employeeSalaries.put('Sam Brown', 55000);
+        employeeSalaries.put('Alice Johnson', 65000);
+
+        if(employeeSalaries.containsKey(employeeName)) {
```

Comment:
You dont need to check for containsKey in this case if you just want to return null, get() will return null if there is no key found.